frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
```

Comment:
We have a custom component Link for that (frontend/src/libs/components/link/link.tsx)

Reply:
i see, let me use it then